src/elements/form/Control.ts

```diff
@@ -95,11 +95,15 @@ import { Helpers } from './../../utils/Helpers';
                     <novo-quick-note *ngSwitchCase="'quick-note'" [formControlName]="control.key" [placeholder]="control.placeholder" [config]="control.config" (change)="modelChange($event)"></novo-quick-note>
                 </div>
             </div>
+            <div class="text-field-character-limit" [attr.data-automation-id]="text-area-character-count" *ngIf="_focused && control.maxlength && (control.controlType=='text-area' || control.controlType=='textbox')">
```

Comment:
It doesnt make sense to me to put it in the error message div, cause its not an error message, its the character count